templates/zerver/help/message-formatting.md

```diff
@@ -0,0 +1,101 @@
+# Message formatting
+
+Zulip uses a variant of GitHub Flavored Markdown (GFM) to allow you to easily format your messages.
+
+## Summary
+
+For a summary of message formatting in Zulip, click the (![A](/static/images/help/formatting.png)) icon located at the bottom left of your messaging box or the cog (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar and choose “Message formatting” from the drop-down menu.
```

Comment:
`(![A](/static/images/help/formatting.png))` should not be surrounded by parentheses, it should just be `...click the ![A](/static/images/help/formatting.png) icon...`